### PR TITLE
fix(node/url): Use Class defined in `node/_error.ts` to throw exception

### PR DESCRIPTION
**Pull request by wafuwafu13**
- Use Class defined in node/_error.ts to throw exception
- Correct Logical OR to Bitwise OR
- Move Comment

### CODE REVIEW

**Comment by kt3k** [comment]
@wafuwafu13 Thank you for your contribution. Can add test cases for this fix?

File: node/_tools/suites/parallel/test-url-fileurltopath.js [hunk 1]
```diff
@@ -10,55 +10,52 @@ const { isWindows } = require('../common');
...
-//     assert.throws(() => url.fileURLToPath('file:///?:/'), {
-//       code: 'ERR_INVALID_FILE_URL_PATH'
-//     });
-//   } else {
-//     assert.throws(() => url.fileURLToPath('file:///a%2F/'), {
-//       code: 'ERR_INVALID_FILE_URL_PATH'
-//     });
-//   }
-// }
+{
```
Comment (wafuwafu13) [inline node/_tools/suites/parallel/test-url-fileurltopath.js:42]: https://github.com/denoland/deno_std/pull/1342/commits/98cc5ae7783adac298ab4522eb2a967587925fba is related.

File: node/url.ts [hunk 2]
```diff
@@ -87,22 +89,22 @@ function getPathFromURLWin(url: URL): string {
...
+      throw new ERR_INVALID_FILE_URL_PATH("must be absolute");
     }
     return pathname.slice(1);
   }
 }
 
 function getPathFromURLPosix(url: URL): string {
   if (url.hostname !== "") {
-    throw new Deno.errors.InvalidData("invalid file url hostname");
+    throw new ERR_INVALID_FILE_URL_HOST(osType);
```
Comment (wafuwafu13) [inline node/url.ts:100]: Ref(argument): https://github.com/nodejs/node/blob/20289e9464ecfca47873e28ff49c2dcca4de304d/lib/internal/url.js#L1449
https://github.com/nodejs/node/blob/20289e9464ecfca47873e28ff49c2dcca4de304d/lib/internal/url.js#L82-L83

**Review by kt3k — APPROVED** [review]
@wafuwafu13 Very nice! LGTM

**Pull request merged**